gapis/api/vulkan/api/synchronization.api

```diff
@@ -166,6 +166,7 @@ cmd VkResult vkWaitForFences(
   @unused VkDevice                  Device
   @unused VkSemaphore               VulkanHandle
   @unused VkQueue                   LastQueue
+  @unused u32                       Submitted
```

Comment:
SubmitCount or something else might be a better name. Submitted by itself sounds like it should be a bool.